src/Monocle/Config.hs

```diff
@@ -201,6 +204,9 @@ type WorkspaceName = IndexName
 
 type WorkspaceStatus = Map WorkspaceName Status
 
+-- | IdentInfo is the Monocle UID and the list of Groups
+type IdentUG = (Text, [Text])
```

Comment:
nit: might be cleaner to use a data record instead of type alias to prevent using the tuple form (like in `getIdentByAliasFromIdents` below)